server/sttp-stub-server/src/main/scala/sttp/tapir/server/stub/package.scala

```diff
@@ -0,0 +1,28 @@
+package sttp.tapir.server
+
+import sttp.client.monad.MonadError
+import sttp.client.monad.syntax._
+import sttp.client.{NothingT, SttpBackend, _}
+import sttp.tapir.client.sttp._
+import sttp.tapir.server.internal.{DecodeInputs, DecodeInputsResult}
+
+package object stub extends SttpStubServer {
+
+  implicit class StubbedEndpoint[I, E, O, F[_]: MonadError](
+      serverEndpoint: ServerEndpoint[I, E, O, Nothing, F]
+  ) {
+    def testUsing(input: I): Option[F[Either[E, O]]] = {
```

Comment:
weren't we suppose to remove this? I think we ended up deciding this doesn't test anything useful? :)

Reply:
This PR is about `testUsing`.